### ISSUE TITLE
make text editor shake as option

### ISSUE THREAD

**Issue by Andrew-liu** [JoelBesada/activate-power-mode#63]
When I use `The activate-power-mode`, it is very excited! But I hope the funciton of shaking is an option.

If you have time to do it, I will appreciate. Thanks

**Comment by markpanado** [comment]
Yeah, agreed! Just installed this now. Shakes hurt my eyes..

**Comment by markpanado** [comment]
Yeah, agreed! Shakes hurt my eyes.. I just installed Atom because of this cool plugin, lol.

**Comment by grjiq** [comment]
I agree.

**Comment by grjiq** [comment]
I change the  intensity in activate-power-mode.coffee to 0. Now it doesn't shake any more.

**Comment by markpanado** [comment]
Thanks for the tip @grjiq it doesn't shake anymore! 

<img width="581" alt="screen shot 2015-12-04 at 1 02 26 am" src="https://cloud.githubusercontent.com/assets/3646720/11567639/e2018b56-9a22-11e5-8ff4-657a76a99e14.png">

**Comment by Andrew-liu** [comment]
**I'm very appreciate @grjiq @markpanado , I have solved the problem!**

```
$  cd ~/.atom/packages/activate-power-mode/lib
$ vim activate-power-mode.coffee
# comment below code and chang it
# intensity = 1 + 2 * Math.random()
intensity = 0
```
finally, reload the `Atom`

**Comment by JoelBesada** [comment]
Fixed in 0.4.0.

**Comment by RyShovan** [comment]
In activate-power-mode version 1.10, I opened the <b>'config-schema.coffee'</b> file and changed the default value of screenShake to <b>false</b>. Restarted atom. and it stopped shaking.
![config-schema](https://cloud.githubusercontent.com/assets/24232212/20681959/e23177e2-b5cb-11e6-8bb5-2f7a82103434.jpg)
